### PR TITLE
Worker queue communication

### PR DESCRIPTION
**Pull request by JulianGindi**

### CODE REVIEW

File: franklin/builder/models.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 from django.db import models
 
 from rest_framework import status
+from .queue import Queue
+
+from .queue import Queue
```
Comment (oudeismetis) [inline franklin/builder/models.py]: `from builder.queue import Queue`

File: franklin/builder/models.py [hunk 2]
```diff
@@ -60,38 +63,20 @@ def is_deployable_event(self, github_event):
...
-            else:
-                logger.error('Builder responded without json')
-        else:
-            logger.error('Bad response from builder.')
-        self.status = self.FAILED
-        self.save()
+        # Push the data into the queue
+        # TODO: Add error handling somehow
+        q = Queue()
+        q.send_message(data)
```
Comment (oudeismetis) [inline franklin/builder/models.py]: In my refactor of api, I have the existing `requests.post()` in a try/catch. Not sure if we need that for the queue, but yea, this code will be easy to merge into my stuff. If my PR goes in soon, I can do the initial merge for you to remove the conflicts if you're still tied up with other tasks.

File: franklin/builder/models.py (hunk 1 above)
Comment (JulianGindi) [inline franklin/builder/models.py]: Well A) I need to remove the duplicate, but B) this style of import is acceptable under [pep8](https://www.python.org/dev/peps/pep-0008/#imports). If we rather stick to the convention of no relative imports, that's fine as long as we all stick with it consistently.

File: franklin/builder/models.py (hunk 2 above)
Comment (JulianGindi) [inline franklin/builder/models.py]: I do need to add some error handling. Will be easier to figure out how to handle errors once I've tested a bit.

**Pull request closed without merging**